fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -182,3 +216,45 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y,
                           event=pd.Series(y).apply(lambda y: _LABEL + "=" + str(y)),
                           **kwargs)
+
+
+class ErrorRateRatio(ConditionalSelectionRate):
+    r"""Implementation of Error Rate Ratio as a moment.
+
```

Comment:
This description does not look correct currently... for example, the definition of `event` looks wrong.

Reply:
This is what `prob_event` says right now
```
The `prob_event` :class:`pandas:pandas.DataFrame` will record the fraction of the samples corresponding to each unique value in the `Y` array.
```
but the event is `_ALL` so it should actually be 1.